CHANGES.txt

```diff
@@ -14,6 +14,8 @@ https://mhammond.github.io/pywin32_installers.html.
 Coming in build 309, as yet unreleased
 --------------------------------------
 
+* Remove temporary `win32com.server.policy` reexports hack (#2344, @Avasam)
```

Comment:
I don't think there's a need for a changelog entry here - we should only add them for things people might possibly notice.

Reply:
Some very old code could reference `win32com.server.policy.DispatcherWin32trace` or `win32com.server.policy.DispatcherTrace`
```suggestion
* Remove temporary `win32com.server.policy` reexports hack (#2344, @Avasam)
  Import `DispatcherWin32trace` and `DispatcherTrace` from `win32com.server.dispatcher` instead.
```

Reply:
Fair enough, a note makes sense then